security-analysis/src/main/java/com/powsybl/security/Security.java

```diff
@@ -174,21 +284,20 @@ public static String printLimitsViolations(List<LimitViolation> violations, Limi
         try (TableFormatter formatter = formatterFactory.create(writer,
                 "",
                 formatterConfig,
-                new Column("Country"),
-                new Column("Base voltage"),
-                new Column("Equipment (" + filteredViolations.size() + ")"),
-                new Column("Violation type"),
-                new Column("Violation name"),
-                new Column("Value")
+                new Column(COUNTRY),
+                new Column(BASE_VOLTAGE),
```

Comment:
The country and the base voltage columns are related to the voltage level. Why this table don't show the voltage level. Move these columns after the voltage level name

Reply:
done